exercises/all-your-base/canonical-data.json

```diff
@@ -16,169 +16,280 @@
     {
       "description": "single bit one to decimal",
       "property": "rebase",
-      "input_base": 2,
-      "input_digits": [1],
-      "output_base": 10,
+      "input": {
+        "input_base": 2,
+        "digits": [
+          1
+        ],
```

Comment:
I don't believe the schema requires arrays to be formatted this way.  It is especially off-putting to see two different style of arrays.  Focus just on creating the "input" object.

Reply:
Gotcha